spirv_glsl.cpp

```diff
@@ -718,6 +719,54 @@ void CompilerGLSL::emit_header()
 			statement("#extension ", ext, " : enable");
 			statement("#endif");
 		}
+		else if (ext == "GL_KHR_shader_subgroup_vote")
+		{
+			declaredKHR_shader_subgroup_vote = true;
+			if (options.vulkan_semantics)
+				statement("#extension GL_KHR_shader_subgroup_vote : require");
+			else
+			{
+				statement("#if defined(GL_ARB_shader_group_vote)");
+				statement("#extension GL_ARB_shader_group_vote : require");
+				statement("#elif defined(GL_NV_gpu_shader5)");
+				statement("#extension GL_NV_gpu_shader5 : require");
+				statement("#elif defined(GL_AMD_gcn_shader)&&defined(GL_AMD_gpu_shader_int64)");
+				statement("#extension GL_AMD_gcn_shader : require");
+				statement("#extension GL_AMD_gpu_shader_int64 : enable"); //needed?
```

Comment:
https://www.khronos.org/registry/OpenGL/extensions/AMD/AMD_gcn_shader.txt

> AMD_gpu_shader_int64 or NV_gpu_shader5 is required.

you need to soft enable both, since spec says "required" and not "implicitly enabled"